src/webview/js/generatedEs3.js

```diff
@@ -520,7 +520,7 @@ documentBody.addEventListener('touchstart', function (e) {
 
   lastTouchPositionX = e.changedTouches[0].pageX;
   lastTouchPositionY = e.changedTouches[0].pageY;
-  lastTouchEventTimestamp = Date.now();
+  touchActive = true;
```

Comment:
Like below, setting `lastTouchEventTimestamp` to `1` here would have the same effect as before since the handler only gets triggered after 500ms in any case.